server/utils/azureAppInsights.ts

```diff
@@ -1,6 +1,12 @@
-import { setup, defaultClient, TelemetryClient, DistributedTracingModes } from 'applicationinsights'
+import { Contracts, setup, defaultClient, TelemetryClient, DistributedTracingModes } from 'applicationinsights'
+import { EnvelopeTelemetry } from 'applicationinsights/out/Declarations/Contracts'
 import applicationVersion from '../applicationVersion'
 
+export type ContextObject = {
+  /* eslint-disable  @typescript-eslint/no-explicit-any */
```

Comment:
i think you should use this form instead

```javascript
// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

because the one you used likely disables the lint for the entire file

Reply:
Oh, missed that, I'll change 👍

Reply:
https://github.com/ministryofjustice/hmpps-incentives-ui/pull/466/commits/c7e0d46a7f9e0d541b335b197004dde4db247cfc